components/web/firefox/Makefile

```diff
@@ -10,96 +10,150 @@
 
-COMPONENT_NAME=		firefox
-IPS_COMPONENT_VERSION=	60.9.0
-COMPONENT_VERSION=	$(IPS_COMPONENT_VERSION)esr
-COMPONENT_REVISION=	2
-COMPONENT_SUMMARY= 	Mozilla Firefox Web browser
-COMPONENT_SRC= 		$(COMPONENT_NAME)-$(IPS_COMPONENT_VERSION)
-COMPONENT_ARCHIVE= 	$(COMPONENT_NAME)-$(COMPONENT_VERSION).source.tar.xz
+ESR = esr
+
+COMPONENT_NAME =	firefox
+COMPONENT_VERSION =	78.9.0
+COMPONENT_REVISION=     0
+COMPONENT_SUMMARY=      Mozilla Firefox Web browser
+COMPONENT_PROJECT_URL =	http://www.mozilla.com/firefox
+COMPONENT_SRC_NAME =	$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_SRC = 	$(COMPONENT_SRC_NAME)
+COMPONENT_ARCHIVE =	$(COMPONENT_SRC_NAME)$(ESR).source.tar.xz
 COMPONENT_ARCHIVE_HASH= \
-	sha256:9f453c8cc5669e46e38f977764d49a36295bf0d023619d9aac782e6bb3e8c53f
-ifdef CANDIDATE_BUILD
-COMPONENT_ARCHIVE_URL= \
-	https://ftp.mozilla.org/pub/firefox/candidates/$(COMPONENT_VERSION)-candidates/build$(CANDIDATE_BUILD)/source/$(COMPONENT_ARCHIVE)
+	sha256:8e03ac1dfc5ac804c8b13a529414a9387e0425e545bb2f4462d74c3175e64864
+ifndef CANDIDATE_BUILD
+MOZILLA_FTP =		http://ftp.mozilla.org/pub/$(COMPONENT_NAME)/releases/$(COMPONENT_VERSION)$(ESR)
 else
-COMPONENT_ARCHIVE_URL= \
-	https://ftp.mozilla.org/pub/firefox/releases/$(COMPONENT_VERSION)/source/$(COMPONENT_ARCHIVE)
+MOZILLA_FTP =		http://ftp.mozilla.org/pub/$(COMPONENT_NAME)/candidates/$(COMPONENT_VERSION)$(ESR)-candidates/build$(CANDIDATE_BUILD)
 endif
+COMPONENT_ARCHIVE_URL =	$(MOZILLA_FTP)/source/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL = http://www.mozilla.com/en-US/firefox/
 COMPONENT_FMRI=		web/browser/firefox
 
-LANG_LIST=	ar bg ca cs da de el es-AR es-CL es-ES et fi fr he hi-IN hr hu id is it ja kk ko lt lv mk nb-NO nl nn-NO pl pt-BR pt-PT ro ru sk sl sq sr sv-SE th tr uk vi zh-CN zh-TW
+LANG_LIST= ach af an ar ast az be bg bn br bs ca-valencia ca cak cs cy da de dsb el en-CA en-GB \
+ en-US eo es-AR es-CL es-ES es-MX et eu fa ff fi fr fy-NL ga-IE gd gl gn gu-IN he hi-IN hr hsb \
+ hu hy-AM ia id is it ja ka kab kk km kn ko lij lt lv mk mr ms my nb-NO ne-NP nl nn-NO oc pa-IN \
+ pl pt-BR pt-PT rm ro ru si sk sl son sq sr sv-SE ta te th tl tr trs uk ur uz vi xh zh-CN zh-TW
+
 ifdef CANDIDATE_BUILD
 LANG_FILES_LOCATION= \
 	http://ftp.mozilla.org/pub/mozilla.org/firefox/candidates/$(COMPONENT_VERSION)-candidates/build$(CANDIDATE_BUILD)/linux-x86_64/xpi/
 else
 LANG_FILES_LOCATION= \
-	http://ftp.mozilla.org/pub/mozilla.org/firefox/releases/$(COMPONENT_VERSION)/linux-x86_64/xpi/
+	http://ftp.mozilla.org/pub/mozilla.org/firefox/releases/$(COMPONENT_VERSION)$(ESR)/linux-x86_64/xpi/
 endif
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+CONFIGURE_DEFAULT_DIRS=no
+
+include $(WS_MAKE_RULES)/common.mk
 
-PATH=$(USRBINDIR.64):$(PATH.illumos)
+#PATH=$(USRBINDIR.64):$(PATH.illumos)
+# /usr/gnu/bin needed to pick up the GNU version of tail and readelf.
+# /usr/sbin needed for dtrace.
+PATH = /usr/gnu/bin:/usr/bin:/usr/sbin
 
```

Comment:
We have PATH.gnu (vs. PATH.illumos) for this so
PATH = $(PATH.gnu)
should be used.